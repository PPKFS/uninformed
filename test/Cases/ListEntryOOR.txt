Home is a room.

The primes are a list of numbers which vary.
The primes are {2, 3, 5, 7, 11}.

When play begins: say "Today's prime is [entry -35 of the primes]."
