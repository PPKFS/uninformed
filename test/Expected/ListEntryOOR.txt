33 words
¶home¶ ¶is¶ ¶a¶ ¶room¶ ¶.¶ ¶|__¶ ¶the¶ ¶primes¶ ¶are¶ ¶list¶ ¶of¶ ¶numbers¶ ¶which¶ ¶vary¶ ¶{¶ ¶2¶ ¶,¶ ¶3¶ ¶5¶ ¶7¶ ¶11¶ ¶}¶ ¶when¶ ¶play¶ ¶begins¶ ¶:¶ ¶say¶ ¶"Today's prime is [entry -35 of the primes]."¶ 
28 distinct words
¶Home¶ ¶home¶ 0a
¶is¶ ¶is¶ 20
¶a¶ ¶a¶ 20
¶room¶ ¶room¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶The¶ ¶the¶ 20
¶primes¶ ¶primes¶ 20
¶are¶ ¶are¶ 20
¶a¶ ¶a¶ 20
¶list¶ ¶list¶ 20
¶of¶ ¶of¶ 20
¶numbers¶ ¶numbers¶ 20
¶which¶ ¶which¶ 20
¶vary¶ ¶vary¶ 20
¶.¶ ¶.¶ 20
¶The¶ ¶the¶ 0a
¶primes¶ ¶primes¶ 20
¶are¶ ¶are¶ 20
¶{¶ ¶{¶ 20
¶2¶ ¶2¶ 20
¶,¶ ¶,¶ 20
¶3¶ ¶3¶ 20
¶,¶ ¶,¶ 20
¶5¶ ¶5¶ 20
¶,¶ ¶,¶ 20
¶7¶ ¶7¶ 20
¶,¶ ¶,¶ 20
¶11¶ ¶11¶ 20
¶}¶ ¶}¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶When¶ ¶when¶ 20
¶play¶ ¶play¶ 20
¶begins¶ ¶begins¶ 20
¶:¶ ¶:¶ 20
¶say¶ ¶say¶ 20
¶"Today's prime is [entry -35 of the primes]."¶ ¶"Today's prime is [entry -35 of the primes]."¶ 20
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
